packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -37,9 +37,21 @@ const AccountSwitcher = props => {
     const [is_deriv_demo_visible, setDerivDemoVisible] = React.useState(true);
     const [is_deriv_real_visible, setDerivRealVisible] = React.useState(true);
     const [is_dmt5_demo_visible, setDmt5DemoVisible] = React.useState(true);
-    const [is_dmt5_real_visible, setDmt5RealVisible] = React.useState(true);
+    const [is_dmt5_real_visible, setDmt5RealVisible] = React.useState(false);
 
     const wrapper_ref = React.useRef();
+    const dmt5_ref = React.useRef(null);
+
+    React.useEffect(() => {
+        if (dmt5_ref.current && is_dmt5_real_visible) {
+            dmt5_ref.current.scrollIntoView({
+                behavior: 'smooth',
+                block: 'end',
+                inline: 'nearest',
+            });
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
We don't need to disable eslint deps check here. Apart from `is_dmt5_real_visible`, there are no other dependencies.

Reply:
removed please approve so we go ahead